setup/tue-functions.bash

```diff
@@ -104,15 +105,15 @@ function _tue-git-branch-clean
         # upon confirmation by the user
         if [ ! $error_code -eq 0 ]
         then
-            unmerged_stale_branches="$unmerged_stale_branches $stale_branch"
+            unmerged_stale_branches="${unmerged_stale_branches:+${unmerged_stale_branches} } $stale_branch"
         fi
     done
 
     # Removal of unmerged stale branches. Not a default operation with the high
     # level command tue-git-branch-clean
     if [ -n "$unmerged_stale_branches" ]
     then
-        unmerged_stale_branches=$(echo "$unmerged_stale_branches" | tr " " "\n")
+        unmerged_stale_branches=$(echo "$unmerged_stale_branches" | sed -e 's/^[[:space:]]*//' | tr " " "\n")
```

Comment:
Is the `sed` command really required because when I last tested this, it worked

Reply:
yep, otherwise there is an empty lime between unmerged stale branches and the actual branches.